[Backport 2.1] Migrating from isolated to execution nodes (PR#300) (#317)
RedHatInsights/red-hat-ansible-automation-platform-documentation: downstream/modules/platform/proc-configure-upgraded-aap.adoc

[id="proc-configure-upgraded-aap_{context}"]

= Configuring your upgraded {PlatformNameShort}

== Configuring {ControllerName} instance groups

> After upgrading your {PlatformName} instance, associate your original instances to its corresponding instance groups by configuring settings in the {ControllerName} UI:

Reviewer: ```suggestion
After upgrading your {PlatformName} instance, associate your original instance with its corresponding instance groups by configuring settings in the {ControllerName} UI:
```
